notNeededPackages.json

```diff
@@ -6335,6 +6335,10 @@
             "libraryName": "superstruct",
             "asOfVersion": "0.8.0"
         },
+        "supports-color": {
+            "libraryName": "supports-color",
+            "asOfVersion": "10.0.0"
```

Comment:
It looks like types are available starting in 9.1 (for index.js only) and completely in 9.2. Would it be better to point there?

Reply:
Seems in v9.x, these types are not mentioned in `package.json`. Marking it as `10.0.0` here may be better.